activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Made the commands inside a `transaction` in a reverted migration also
+    run reverted.
+
+    Fixes #21484.
```

Comment:
I don't think this is enough to "fix" the referenced issue. Basically it's talking about a custom block helper that did not get reverted in reverse order. I don't think that's something we intend to address.

> See #21484.